guides/common/assembly_installing-satellite-server-connected.adoc

```diff
@@ -0,0 +1,86 @@
+[id="installing-satellite-server-connected"]
+= Installing {ProjectServer}
+
+ifeval::["{build}" == "satellite"]
+When you install {ProjectServer} from a connected network, you can obtain packages and receive updates directly from the Red Hat Content Delivery Network.
+
+Use the following procedures to install {ProjectServer}, perform the initial configuration, and import subscription manifests.
+
+ifeval::["{build}" != "foreman-deb"]
+Before you continue, consider which manifests are relevant for your environment. For more information on manifests, see link:{BaseURL}content_management_guide/managing_subscriptions/[Managing Subscriptions] in the _Content Management Guide_.
+endif::[]
+
+
+NOTE: You cannot register {ProjectServer} to itself.
+endif::[]
```

Comment:
There is `ifeval::["{build}" != "foreman-deb"]` statement inside of ` ifeval::["{build}" == "satellite"]`.  This does not make sense because the content inside of the first ` ifeval::["{build}" == "satellite"]` will be shown in satellite build only.

Reply:
Oops! That was an error :smile: